src/averageAges.js

```diff
@@ -15,6 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let men = people.filter(person => person.sex === 'm');
+
+  century && (men = men
+    .filter(person => Math.ceil(person.died / 100) === century));
```

Comment:
```suggestion
  const men = people.filter(person =>  century 
    ? person.sex === 'm'
    : person.sex === 'm' && Math.ceil(person.died / 100) === century
  );
```

Reply:
the same here , is it bad ? because my code follows the instruction 
// replace `if ()` statement with &&, || or ?:
  // without nesting

Reply:
and maybe you meant that   
const men = people.filter(person => century
    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
    : person.sex === 'm'
  );